app/cdap/components/GridTable/index.tsx

```diff
@@ -138,7 +138,10 @@ export default function GridTable() {
     }
   };
 
-  // ------------@createMissingData Function is used for preparing data for second row of Table which shows Missing/Null Value
+  const setOpenColumnViewHandler = () => {
```

Comment:
why is open part of the function name, arent we using same function to close the panel as well

also, why do we need function when we have single statement inside
can u just move the setOpenColumnView into JSX plz and remove the function